### DISCUSSION TITLE
Any update on sudo?

### DISCUSSION THREAD

**Discussion by AvogatoWizardWhisker** [microsoft/sudo#94] [General]
It's been a few months since `sudo` for windows was released to GitHub and has been open sourced, but we haven't heard anything since. I'm wondering what they're cooking 👀.

**Comment by zadjii-msft** [comment]
Unfortunately, things have been terribly boring, that's why you haven't heard much. Our big-picture next goal is to build out an `msi` installer so folks can get sudo on Windows 10 et.al. But there are some internal-to-Microsoft engineering things that we need to get sorted first (namely, updating our cargo feed to use a azure artifacts one rather than the public cargo feed, but also there are ~ r e a s o n s ~ why that's hard for an OSS project.....)

Basically, we're just waiting on some other teams to do boring behind the scenes work. Sorry for not sharing an update sooner!

**Reply by roke-julian-lockhart** [reply]
> https://github.com/microsoft/sudo/discussions/94#discussioncomment-10204707

@zadjii-msft, why MSI, rather than MSIX?

**Reply by zadjii-msft** [reply]
Eh, I've had more than my fair share of weird edge cases with app execution aliases with `wt.exe`, and I didn't really want to complicate that further with cross-elevation scenarios. `sudo` isn't terribly complicated - an `exe`, and some GPO policies. An MSIX seems a little like overkill.

**Reply by zadjii-msft** [reply]
(admittedly also, my baby was born earlier than expected this week, so I wasn't able to get to this before my paternity started. I won't be able to get to this soon, but I'd happily be able to review a contribution that puts one together)

**Reply by roke-julian-lockhart** [reply]
> Eh, I've had more than my fair share of weird edge cases with app execution aliases with wt.exe, and I didn't really want to complicate that further with cross-elevation scenarios. sudo isn't terribly complicated - an exe, and some GPO policies. An MSIX seems a little like overkill.

@zadjii-msft, I was merely interested because it'd be a nicer solution for https://github.com/microsoft/sudo/issues/57#issue-2138485465, since it integrates better with PowerShell.

> (admittedly also, my baby was born earlier than expected this week, so I wasn't able to get to this before my paternity started. I won't be able to get to this soon, but I'd happily be able to review a contribution that puts one together)

I know hearing congratulations from a stranger is fairly meaningless, but the thought is there.

**Reply by GitMurf** [reply]
> admittedly also, my baby was born earlier than expected this week, so I wasn't able to get to this before my paternity started.

Congratulations @zadjii-msft ! I currently have three under 6 years old 🤣 so I know how parenthood goes! Enjoy and best of luck! `sudo family first = true` 😉
